server.py

```diff
@@ -322,11 +330,10 @@ def start_server(self):
         if Config.enable_socket_io_api:
             logging.info('Socket.io API enabled')
             self.sio = socketio.AsyncServer(async_mode='aiohttp',
-                                            ping_timeout=60,
-                                            ping_interval=25,
-                                            cookie=None,
-                                            cors_allowed_origins='*',
-                                            logger=False)
+                                            # ping_timeout=60,
+                                            # ping_interval=25,
+                                            # cookie=None,
```

Comment:
Why are these lines commented out?

Reply:
Short reminders for myself. I will clean it up before releasing a new version